### PR TITLE
Add support to build sample python package with docker

### PR DESCRIPTION
**Pull request by elainechien**
This allows the python package to be built in an isolated docker container so that during build the package does not modify the host machine. Docker is not truly sandboxed but is fine for the purposes of the test package.

#1001

### CODE REVIEW

File: sample_packages/sample_python_package/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM python:3.9
```
Comment (maxfisher-g) [inline sample_packages/sample_python_package/Dockerfile:1]: We should pin the SHA256 hash of the container here

**Review by maxfisher-g — APPROVED** [review]
Thanks Elaine, just one minor comment

File: sample_packages/sample_python_package/Dockerfile (hunk 1 above)
Comment (elainechien) [reply sample_packages/sample_python_package/Dockerfile]: Do you mean the sha for the image? Updated the PR. I took the latest python3.9 sha from here: https://hub.docker.com/layers/library/python/3.9/images/sha256-edee3d665aba1d84f5344eca825d0de34b38dbf77a776cafd9df65c67e240866?context=repo&tab=vulnerabilities

Comment (maxfisher-g) [reply sample_packages/sample_python_package/Dockerfile]: yes, thanks!

**Review by maxfisher-g — APPROVED** [review]

**Pull request merged**